### PR TITLE
[pom repository] support multiple gavs in the revision field for easier configuration

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

File: biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java [hunk 1]
```diff
@@ -51,9 +54,9 @@
 	private String				name;
 	private Reporter			reporter			= new Slf4jReporter(BndPomRepository.class);
 	private InnerRepository	repoImpl;
-	private Revision			revision;
+	private Set<Revision>		revisions;
```
Comment (bjhargrave) [inline biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java]: Does this need to be a Set? Are we worried about dups? We need a well defined iteration order. Use List.

File: biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java [hunk 2]
```diff
@@ -51,9 +54,9 @@
...
 	private BridgeRepository	bridge;
-	private URI					pomFile;
+	private Set<URI>			pomFiles;
```
Comment (bjhargrave) [inline biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java]: See above. Use List.

File: docs/_plugins/pomrepo.md [hunk 3]
```diff
@@ -67,6 +67,8 @@ TODO: The syntax of the query part might change
 |                  |               |overridden with the `maven.repo.local` System property as in Maven| 
 | `revision`       | `GAV`         |The Maven coordinates. The GAV will be searched in the normal way.|
```
Comment (bjhargrave) [inline docs/_plugins/pomrepo.md]: You forget to delete the old lines.

File: biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/PomRepository.java [hunk 4]
```diff
@@ -23,53 +25,55 @@
 	static final String		BND_MAVEN_EXCEPTION_ATTRIBUTE	= "exception";
 	static final String		BND_MAVEN_ARCHIVE_ATTRIBUTE		= "archive";
 	static final String		BND_MAVEN_REVISION_ATTRIBUTE	= "revision";
-	final Revision			revision;
-	final URI				revisionUrl;
+	final Set<Revision>	revisions;
+	final Set<URI>		uris;
```
Comment (bjhargrave) [inline biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/PomRepository.java]: See above, Both should be List.

File: biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/Traverser.java [hunk 5]
```diff
@@ -43,35 +44,29 @@
 	static final String						ROOT		= "<>";
 	final ConcurrentMap<Archive,Resource>	resources	= new ConcurrentHashMap<>();
 	final Executor							executor;
-	final Collection<Revision>				revisions;
-	final URI								resource;
+	final Set<Revision>						revisions;
+	final Set<URI>							uris;
```
Comment (bjhargrave) [inline biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/Traverser.java]: See above, both should be List.

**Comment by rotty3000** [comment]
hey @bjhargrave I have updated this.

**Pull request merged**